WDAC-Policy-Wizard/app/src/Helper.cs

```diff
@@ -511,42 +511,26 @@ public static int CompareVersions(string minVersion, string maxVersion)
             return 0;
         }
 
+        /// <summary>
+        /// Validates the custom path rule against the requirements for path rules in WDAC
+        /// Updated 22H2: path rules now support multiple wildcards throughout the path
+        /// </summary>
+        /// <param name="customPath"></param>
+        /// <returns>True if valid path rule in WDAC. False, otherwise.</returns>
         public static bool IsValidPathRule(string customPath)
         {
             // Check for at most 1 wildcard param (*)
-            if (customPath.Contains("*"))
```

Comment:
Do you really want to remove this altogether or add a non-fatal warning that the rule will only be valid on Win11? Downside of that is obviously we'd need to remove it once Win10 backport for wildcards has been completed.